### ISSUE TITLE
Lost actions on last version

### ISSUE THREAD

**Issue by mlopezsti** [eminiarts/nova-tabs#228]
When install the last version, lost actions on detail.

**Comment by marcfilipan** [comment]
Could you possibly post the action that you use?

**Comment by mlopezsti** [comment]
what action?, In detail view lost all list actions.

version 2.0.7:

![image](https://user-images.githubusercontent.com/67266571/175968906-7931276a-787f-4f93-9d21-8bc7efd4e2e5.png)

![image](https://user-images.githubusercontent.com/67266571/175969095-56a2ddbc-0b1d-409d-8da7-a2d5cb9e5c15.png)

version 2.0.9:

![image](https://user-images.githubusercontent.com/67266571/175968444-cb56474e-82be-4033-9e3b-156d5025c4ea.png)

**Comment by marcfilipan** [comment]
Call the withToolbar method on Tabs

**Comment by marcfilipan** [comment]
Actions never stopped working for me, but perhaps try 2.1.0 and see if they show up for you again

<img width="2677" alt="image" src="https://user-images.githubusercontent.com/1815234/176184876-e5195183-c406-4c8c-8c79-396663609307.png">

**Comment by mlopezsti** [comment]
I can understand add for example withoutToolbar to hidden action tollbar. Not the other way round

**Comment by marcfilipan** [comment]
Acting entitled and annoyed won't get you anywhere.

Post the code of your resource so I can look what you're doing wrong.

**Comment by mlopezsti** [comment]
I publish my code.

on first post

**Comment by mlopezsti** [comment]
Only read and I wrote my code using your wiki.

**Comment by marcfilipan** [comment]
> Only read and I wrote my code using your wiki.

Let me rephrase, post the actual code, so I can correct it for you, as you're not calling the withToolbar method.

**Comment by mlopezsti** [comment]
```php
         Tabs::make('XXXX', [
          Tab::make('XXXX Info', [
             
          ]),
          Tab::make('XXX', [
              
          ]),
          Tab::make('XXXX', [
             
          ]),
          Tab::make('YYYYY', [
             
          ]),
          Tab::make('YYYY', [
              HasMany::make('YYYY', 'YYYY')
          ]),
          Tab::make('YYYY', [
              HasMany::make('YYYYY')
          ]),
          Tab::make('AAAAAA', [
              HasMany::make('AAAAA', 'aaaaa', 'App\Nova\AAAaaa')
          ]),
          Tab::make('UUUUU', [
              HasMany::make('UUUUU', 'UUUU', '\App\Nova\UUUUU'),
          ]),
          Tab::make('Actions', [
              $this->actionfield(), // Add Actions whererver you like.
          ]),
      ]),
```

**Comment by marcfilipan** [comment]
```
         Tabs::make('XXXX', [
          Tab::make('XXXX Info', [
             
          ]),
          Tab::make('XXX', [
              
          ]),
          Tab::make('XXXX', [
             
          ]),
          Tab::make('YYYYY', [
             
          ]),
          Tab::make('YYYY', [
              HasMany::make('YYYY', 'YYYY')
          ]),
          Tab::make('YYYY', [
              HasMany::make('YYYYY')
          ]),
          Tab::make('AAAAAA', [
              HasMany::make('AAAAA', 'aaaaa', 'App\Nova\AAAaaa')
          ]),
          Tab::make('UUUUU', [
              HasMany::make('UUUUU', 'UUUU', '\App\Nova\UUUUU'),
          ]),
          Tab::make('Actions', [
              $this->actionfield(), // Add Actions whererver you like.
          ]),
      ])->withToolbar(),
```
